src/rust/include/rust/orchard.h

```diff
@@ -108,6 +108,20 @@ void orchard_batch_add_bundle(
 ///
 /// Returns false if any item in the batch is invalid.
 bool orchard_batch_validate(const OrchardBatchValidatorPtr* batch);
+
+/// Returns whether the Orchard bundle is present and outputs
+/// are enabled.
+bool orchard_bundle_outputs_enabled(const OrchardBundlePtr* bundle);
+
+/// Returns whether the Orchard bundle is present and spends
+/// are enabled.
+bool orchard_bundle_spends_enabled(const OrchardBundlePtr* bundle);
+
+/// Returns whether all actions contained in the Orchard bundle
+/// can be decrypted with the all-zeros OVK. Returns `true`
+/// if no Orchard actions are present.
+bool orchard_bundle_has_valid_coinbase_outputs(const OrchardBundlePtr* bundle);
```

Comment:
Consider renaming this to `orchard_bundle_coinbase_outputs_are_valid` (to avoid the implication that there must be any such outputs).